### PR TITLE
[15.0][MIG] product_abc_classification_sale: Migration to version 15.0

### PR DESCRIPTION
**Pull request by stefan-tecnativa**
cc @Tecnativa TT38329

@chienandalu @CarlosRoca13 please review!

This PR depends on #1235

### CODE REVIEW

**Comment by chienandalu** [comment]
please review @pilarvargas-tecnativa

**Comment by pilarvargas-tecnativa** [comment]
@stefan-tecnativa please, can you push it again so I can test it?

File: product_abc_classification_sale/tests/test_sale_product_classification_common.py [hunk 1]
```diff
@@ -80,12 +80,13 @@ def _create_sale(cls, date, partner, products, qty=1000):
...
         sale_order = sale_form.save()
-        sale_order.action_confirm()
+        sale_order.action_done()
         # Force the date so to reproduce the calendar
         sale_order.date_order = date
 
     def _test_product_classification(self, classifications):
         """Helper to assert classifications in batch. It's expected to come
         as dict of key product record and value string classification"""
+
```
Comment (CarlosRoca13) [inline product_abc_classification_sale/tests/test_sale_product_classification_common.py]: Remove this empty line

**Review by CarlosRoca13 — APPROVED** [review]
Just a minor comment

**Comment by pilarvargas-tecnativa** [comment]
One small suggestion, regenerate the readme to include the changes :)

File: product_abc_classification_sale/tests/test_sale_product_classification_common.py (hunk 1 above)
Comment (stefan-tecnativa) [reply product_abc_classification_sale/tests/test_sale_product_classification_common.py]: Thanks for the suggestion

**Review by pilarvargas-tecnativa — APPROVED** [review]
Revised code and functionality :)

**Pull request merged**